app.py

```
@@ -46,464 +37,256 @@ def get_env_var(var_name):
+                    INSERT INTO attachments (email_id, filename, content)
+                    VALUES (:email_id, :filename, :content);
+                """)
+                conn.execute(insert_attachment_query, {
+                    'email_id': email_id,
+                    'filename': sanitize_filename(attachment.name),
+                    'content': attachment.content
+                })
 
-    return base_name
+        trans.commit()
+        return email_id
```

Ruff (0.13.0)

Consider moving this statement to an `else` block

(TRY300)